packages/angular_devkit/build_angular/src/webpack/plugins/devtools-ignore-plugin.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Compilation, Compiler } from 'webpack';
+
+// Following the naming conventions from
+// https://sourcemaps.info/spec.html#h.ghqpj1ytqjbm
+const IGNORE_LIST = 'x_google_devtoolsIgnore';
+
+const PLUGIN_NAME = 'devtools-ignore-plugin';
+
+interface Options {
+  vendors?: false | { test: RegExp };
+  runtime?: false | { test: RegExp };
+}
+/**
+ * This plugin adds a field to source maps that identifies which sources are
+ * vendored or runtime-injected (aka third-party) sources. These are consumed by
+ * Chrome DevTools to automatically ignore-list sources.
+ */
+export class DevToolsIgnorePlugin {
+  options: Options;
+
+  constructor(options: Options) {
+    this.options = options;
+  }
+
+  apply(compiler: Compiler) {
+    const { SourceMapSource } = compiler.webpack.sources;
+
+    compiler.hooks.compilation.tap(PLUGIN_NAME, (compilation) => {
+      compilation.hooks.processAssets.tap(
+        {
+          name: PLUGIN_NAME,
+          stage: Compilation.PROCESS_ASSETS_STAGE_DEV_TOOLING,
+          additionalAssets: true,
+        },
+        (assets) => {
+          if (!this.options.vendors || !this.options.runtime) {
```

Comment:
Should this be `&&` instead of `||`? IE: if either of them is truthy we should not exit.

Reply:
You're right, I was just looking at this as well. This can be either or, will make the change.